mypy.ini

```diff
@@ -1,6 +1,7 @@
 [mypy]
 python_version = 3.9
 check_untyped_defs = True
+exclude = venv
```

Comment:
For some reason mypy was flagging errors with venv code, so I explicitly excluded it.